tests/lean/interactive/jumpMutual.lean

```diff
@@ -3,7 +3,6 @@ mutual
 def h (x : Nat) : Nat :=
   match x with
   | 0 => 1
-                     --v textDocument/definition
```

Comment:
I'm not sure if there was a particular reason this was kept here but it definitely doesn't have the effect of testing go-to-definition on `h` (in this PR it produces `-32603: unknown constant 'Lean.Elab.Term.elabMatch'` which breaks the test).